packages/bvaughn-architecture-demo/components/sources/PointPanel.module.css

```diff
@@ -80,6 +84,36 @@
   width: 100%;
 }
 
+.Prev {
+  background-color: white;
+  width: 14px;
+  height: 14px;
+  border-radius: 7px;
+}
+
+.Next {
+  background-color: white;
```

Comment:
btw any colors need to go into theme variables or they'll break things for one theme or the other.